probe/probe.go

```diff
@@ -0,0 +1,315 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2021 Authors of KubeArmor
+
+package probe
+
+
+import (
+	"fmt"
+	"syscall"
+    "strings"
+	"golang.org/x/mod/semver"
+    "log"
+    "github.com/fatih/color"
+    "os"
+    "io"
+    "time"
+    "github.com/kubearmor/kubearmor-client/k8s"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/tools/remotecommand"
+	"context"
+	corev1 "k8s.io/api/core/v1"
+	"github.com/kubearmor/kubearmor-client/deployment"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var white = color.New(color.FgWhite)
+var boldWhite = white.Add(color.Bold)
+var karmorprobe = "karmor-probe"
```

Comment:
Let's import the name declared in deployment package? WDYT? Could avoid duplication if we need to change names sometime.